doc/documentation.md

```diff
@@ -55,18 +55,3 @@ The Web Site is hosted by GitHub Pages from the docs/ directory of the gh-pages
 
 When updates to LinkChecker are pushed, the web site is built and published
 automatically by a GitHub action ``.github/workflows/publish-pages.yml``.
-
-For information, a manual process to build and publish the web site would look like:
```

Comment:
what would be the manual process now? is it documented somewhere? if not, why shouldn't it be? :)

Reply:
I haven't worked it out! Well, I think it needs 2 repositories but I don't have the exact steps.

The section above does document how to create the HTML, maybe `setup.py build` should be added.

Reply:
well maybe it's fine to remove the outdated docs in the meantime, i retract my objection but it would be nice to see how this works outside of CI.

Reply:
By "2 repositories" do you mean two checkouts of the same repository?

I've used [ghp-import](https://pypi.org/project/ghp-import/) in the past, it works great.  No extra checkouts needed, just a docs build in a local .gitignore'd directory that gets imported into the root of the gh-pages branch by the ghp-import script.

But I don't think we'll need that if we have the GH action.

Reply:
I have added the setup.py build step, in its own section because it is needed for man pages too.
That does mean our steps are recorded, even if there isn't a recipe for a manual publish. At the moment any merge will break the web site so best get this done.

Yes, two local ones. ghp-import looks interesting, although with the action there is no need to do anything, as long as it works...